react/OrderTotal.tsx

```diff
@@ -18,26 +20,33 @@ const OrderTotal: FC = () => {
   const { items, totals, value: totalValue } = useOrder()
   const handles = useCssHandles(CSS_HANDLES)
   const numItems = items.reduce((acc, item) => acc + item.quantity, 0)
+  const [newTotals, taxes] = getTotals(totals)
 
   return (
     <div className={`${handles.totalListWrapper} flex-l justify-end w-100`}>
       <ul
         className={`${handles.totalList} list pa0 mt8 w-100 w-60-l c-muted-1`}
       >
-        {totals.map(total => (
-          <li
-            className={`${handles.totalListItem} pv3 flex justify-between items-center`}
-            key={total.id}
-          >
-            <span className={`${handles.totalListItemLabel}`}>
-              <TranslateTotalizer totalizer={total} />
-              {total.id === 'Items' && ` (${numItems})`}
-            </span>
-            <span className={`${handles.totalListItemValue} c-on-base`}>
-              <FormattedPrice value={total.value} />
-            </span>
-          </li>
-        ))}
+        {newTotals.map(total => {
+          if (total.value === 0) return null
+          return (
+            <li
+              className={`${handles.totalListItem} pv3 flex justify-between items-center`}
+              key={total.id}
+            >
+              <span className={`${handles.totalListItemLabel} flex`}>
+                <TranslateTotalizer totalizer={total} />
+                {total.id === 'Items' && ` (${numItems})`}
+                {total.id === 'Tax' && taxes.length > 0 && (
+                  <TaxInfo taxesTotals={taxes} className="pl2 pt1" />
```

Comment:
I would wrap it with a div instead of passing these classes. I think the components itself doesn't need to know about its outer spacings.

```suggestion
                  <div className="ml2 mt1">
                      <TaxInfo taxesTotals={taxes} />
                   </div>
```

Reply:
I agree again